mods/ts/rules/civilian-infantry.yaml

```diff
@@ -270,13 +272,16 @@ VISC_LRG:
 		HP: 500
 	Armament:
 		Weapon: SlimeAttack
+		FireDelay: 10
```

Comment:
Sorry for asking the obvious, but I guess you did make sure you updated to the latest version (with the map fixes) of this branch before creating that replay?